capellambse/model/layers/oa.py

```diff
@@ -31,15 +31,15 @@ class OperationalActivity(fa.AbstractFunction):
     exchanges = c.DirectProxyAccessor(
         fa.FunctionalExchange, aslist=c.ElementList
     )
-    owner: c.Accessor[Entity]
 
-    @property
-    def inputs(self) -> c.ElementList[fa.FunctionalExchange]:
-        return self._model.oa.all_activity_exchanges.by_target(self)
+    inputs = c.ReferenceSearchingAccessor(
+        fa.FunctionalExchange, "target", aslist=c.MixedElementList
+    )
+    outputs = c.ReferenceSearchingAccessor(
+        fa.FunctionalExchange, "source", aslist=c.MixedElementList
+    )
```

Comment:
We're only looking at `FunctionalExchange` instances, so these should use the normal `ElementList`.